geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/distributed/WaitUntilFlushedFunction.java

```diff
@@ -85,8 +72,8 @@ public void execute(FunctionContext context) {
       }
 
     } else {
-      throw new IllegalArgumentException(
-          "The AEQ does not exist for the index " + indexName + " region " + region.getFullPath());
+      resultSender.sendException(new IllegalStateException(
```

Comment:
I think we should just throw the IllegalStateException, that way it will propagate to the user and let them know that an AEQ does not exist for the index.